pyrobosim/pyrobosim/gui/main.py

```diff
@@ -372,11 +422,41 @@ def on_close_click(self) -> None:
         elif (robot is None) and self.goal_textbox.text():
             self.world.close_location(self.goal_textbox.text())
 
-    def on_collision_polygon_toggle_click(self) -> None:
-        """Callback to toggle collision polygons."""
+    def on_toggle_collision_polygons(self, state: int) -> None:
+        """Callback to toggle collision polygons.
+
+        :param state: Integer state of the checkbox (0=unchecked, 2=checked)"""
```

Comment:
for all of these, can you make the end of the quotes `"""` be in a new line?

That's my only remaining comment -- all works great!